common/djangoapps/xblock_django/user_service.py

```diff
@@ -26,12 +26,25 @@ def __init__(self, django_user, **kwargs):
         if self._django_user:
             self._django_user.user_is_staff = kwargs.get('user_is_staff', False)
 
+    @property
+    def django_user(self):
+        """
+        Return the django user object.
+        """
+        return self._django_user
```

Comment:
Understood, and done with 6eca330 .

Though I'm not clear on when the `self.scope_ids.user_id` should be used instead of the `user_service..opt_attrs[ATTR_KEY_USER_ID]`?